#!/usr/bin/python
PACKAGE = 'recognition'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add('embedder_folder_path',     str_t,   0, 'the path to the face detector modelfile', '/data/face_embedder')

exit(gen.generate(PACKAGE, 'recognition', 'FaceEmbedding'))

